npk/src/manifest.rs

```diff
@@ -204,6 +204,13 @@ pub struct Manifest {
     )]
     #[serde(default)]
     pub mounts: HashMap<PathBuf, Mount>,
+    /// String containing capability names to give to
+    /// new container
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub capability_str: Option<String>,
+    /// String containing group names to give to new container
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub supplgroups_str: Option<String>,
```

Comment:
This should be a `Option<Vec<String>>`

Reply:
Can you (or someone else), then provide a fix ?

I do not have the Rust knowledge to do what you want. I am just following the examples of what is already there.

Reply:
Sure. I'm currently a little bit short on time but I think that's just a finger exercise for @marcmo or @alfonsoros88. We can ask in the SU tomorrow.